bug: Scanner not included by SPM

### Did you check existing issues?

- [X] I have read all the [tree-sitter docs](https://tree-sitter.github.io/tree-sitter/using-parsers) if it relates to using the parser
- [X] I have searched the existing issues of tree-sitter-c

### Tree-Sitter CLI Version, if relevant (output of `tree-sitter --version`)

_No response_

### Describe the bug

Using *tree-sitter-ruby* from SPM (Swift Package Manager) results in build errors:

```text
Undefined symbols for architecture arm64:
  "_tree_sitter_ruby_external_scanner_create", referenced from:
      _tree_sitter_ruby.language in TreeSitterRuby.o
  "_tree_sitter_ruby_external_scanner_deserialize", referenced from:
      _tree_sitter_ruby.language in TreeSitterRuby.o
  "_tree_sitter_ruby_external_scanner_destroy", referenced from:
      _tree_sitter_ruby.language in TreeSitterRuby.o
  "_tree_sitter_ruby_external_scanner_scan", referenced from:
      _tree_sitter_ruby.language in TreeSitterRuby.o
  "_tree_sitter_ruby_external_scanner_serialize", referenced from:
      _tree_sitter_ruby.language in TreeSitterRuby.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Due to the fact that *scanner.c* is not included as source as specified in *Package.swift*:

https://github.com/tree-sitter/tree-sitter-ruby/blob/a66579f70d6f50ffd81a16fc3d3358e2ac173c88/Package.swift#L40-L43


### Steps To Reproduce/Bad Parse Tree

1. Create an application/tool in Xcode (or create a package on the command line using SPM).
2. Add *tree-sitter* and *tree-sitter-ruby* libraries as dependencies.
3. Call `tree_sitter_ruby()` in Swift (not related to the error, but compiling an empty project may skip compiling *tree-sitter* and *tree-sitter-ruby*)
4. Encounter build failure with exact error message in bug description.

### Expected Behavior/Parse Tree

*tree-sitter-ruby* compiles successfully as a self-contained library.

### Repro

```C
import TreeSitterRuby

_ = tree_sitter_ruby()
```

--- a/Package.swift
+++ b/Package.swift
@@ -4,10 +4,10 @@ import PackageDescription
 let package = Package(
     name: "TreeSitterRuby",
     products: [
-        .library(name: "TreeSitterRuby", targets: ["TreeSitterRuby"]),
+        .library(name: "TreeSitterRuby", targets: ["TreeSitterRuby"])
     ],
     dependencies: [
-        .package(url: "https://github.com/ChimeHQ/SwiftTreeSitter", from: "0.8.0"),
+        .package(url: "https://github.com/ChimeHQ/SwiftTreeSitter", from: "0.8.0")
     ],
     targets: [
         .target(
@@ -39,7 +39,7 @@ let package = Package(
             ],
             sources: [
                 "src/parser.c",
-                // NOTE: if your language has an external scanner, add it here.
+                "src/scanner.c",
             ],
             resources: [
                 .copy("queries")
@@ -54,7 +54,7 @@ let package = Package(
                 "TreeSitterRuby",
             ],
             path: "bindings/swift/TreeSitterRubyTests"
-        )
+        ),
     ],
     cLanguageStandard: .c11
 )

